frontend/src/pages/modelServing/screens/projects/NIMServiceModal/NoAuthAlert.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Alert, AlertActionCloseButton, StackItem } from '@patternfly/react-core';
+
+export const NoAuthAlert: React.FC<{ setAlertVisible: (isVisible: boolean) => void }> = ({
```

Comment:
Having an `onClose` callback makes more sense because the boolean param is never set to anything other than `false`.
```suggestion
export const NoAuthAlert: React.FC<{ onClose: () => void }> = ({
```